SpatialGDK/Source/SpatialGDKEditor/Public/EditorExtension/LBStrategyEditorExtension.h

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+
+class UAbstractLBStrategy;
+class FLBStrategyEditorExtensionManager;
+struct FWorkerTypeLaunchSection;
+
+class FLBStrategyEditorExtensionInterface
+{
+private:
+	friend FLBStrategyEditorExtensionManager;
+	virtual bool GetDefaultLaunchConfiguration_Virtual(const UAbstractLBStrategy* Strategy, FWorkerTypeLaunchSection& OutConfiguration, FIntPoint& OutWorldDimensions) const = 0;
+};
+
+template <typename StrategyImpl, typename Implementation>
+class FLBStrategyEditorExtensionTemplate : public FLBStrategyEditorExtensionInterface
+{
+public:
+	using ExtendedStrategy = StrategyImpl;
+private:
+
```

Comment:
Nit - spacing is inconsistent around access modifiers. I think we generally stick to a space above one but not below.